releasenotes/notes/fix_readiness-c0e1f30b49efe5f5.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes the `/ready` health endpoint on the cluster-agent.
```

Comment:
Could you add a bit more details here (slightly too generic for the CHANGELOG).

Reply:
I just added some extra explanation.